### PR TITLE
fix minicpmo4.5: fix attn_mask in vit attn && fix resampler pos_emb i…

### PR DESCRIPTION
**Pull request by tc-mb**
We verified the accuracy of the model and found two accuracy issues that we wanted to merge to achieve perfect performance for the model.

1. Attn_mask is not entered in ViT's Attention.
2. pos_emb in 3D-Resampler is added incorrectly on k / v.

### CODE REVIEW

File: vllm/model_executor/models/idefics2_vision_model.py [hunk 1]
```diff
@@ -370,15 +400,42 @@ def forward(
         patch_attention_mask: torch.BoolTensor | None = None,
         tgt_sizes: torch.IntTensor | None = None,
     ) -> torch.Tensor:
+        batch_size = pixel_values.size(0)
+        if patch_attention_mask is None:
+            patch_attention_mask = torch.ones(
```
Comment (DarkLight1337) [inline vllm/model_executor/models/idefics2_vision_model.py:408]: Is it necessary to initialize patch_attention_mask? Can we change L425-L433 like
```py
        if patch_attention_mask is None:
            attention_mask = None
        else:
            patch_attention_mask = torch.ones(...)
            flat_patch_mask = patch_attention_mask.view(batch_size, -1)

            # Additive mask: masked positions receive a large negative value.
            # Shape: (B, 1, 1, L) broadcastable to (B, H, Q, K).
            min_val = torch.finfo(hidden_states.dtype).min
            attention_mask = (~flat_patch_mask).to(dtype=hidden_states.dtype) * min_val
            attention_mask = attention_mask[:, None, None, :]

Comment (tc-mb) [reply vllm/model_executor/models/idefics2_vision_model.py:408]: ok, fixed

Comment (DarkLight1337) [reply vllm/model_executor/models/idefics2_vision_model.py:408]: I mean, can we avoid initializing `patch_attention_mask` unless an explicit value is passed?

Comment (tc-mb) [reply vllm/model_executor/models/idefics2_vision_model.py:408]: @DarkLight1337  Sorry for the late reply due to Chinese New Year holiday.
The patch_attention_mask initialization here is necessary in the original code, patch_attention_mask was not initialized when None is passed, but self.embeddings() directly iterates over it and uses it for indexing, which would cause runtime errors.

**Review by DarkLight1337 — APPROVED** [review]
Sorry forgot about this
